README.md

```diff
@@ -62,35 +62,42 @@ Select *Stash Pull Request Builder* then configure:
 
 ## Building the merge of Source Branch into Target Branch
 
-You may want Jenkins to build the merged PR (that is the merge of `sourceBranch` into `targetBranch`) to catch any issues resulting from this. To do this change the Branch Specifier from `origin/pr/${pullRequestId}/from` to `origin/pr/${pullRequestId}/merge`
+### Letting Stash do the merge
 
-If you are building the merged PR you probably want Jenkins to do a new build when the target branch changes. There is an advanced option in the build trigger, "Rebuild if destination branch changes?" which enables this.
+You may want Jenkins to build the merged PR (that is the merge of `sourceBranch` into `targetBranch`) to catch any issues resulting from this. To do this, change the Branch Specifier from `origin/pr/${pullRequestId}/from` to `origin/pr/${pullRequestId}/merge`
+
+If you are building the merged PR, you probably want Jenkins to do a new build when the target branch changes. There is an advanced option in the build trigger, "Rebuild if destination branch changes" which enables this.
 
 You probably also only want to build if the PR was mergeable and always without conflicts. There are advanced options in the build trigger for both of these. 
 
-**NOTE: *Always enable `Build only if Stash reports no conflicts` if using the merge RefSpec!*** This will make sure the lazy merge on stash has happened before the build is triggered.
+**NOTE: *Always enable `Build only if Stash reports no conflicts` if using the merge RefSpec!*** This will make sure the lazy merge on Stash has happened before the build is triggered.
+
+### Merging locally, variant 1
+
+If you don't want to use the lazy merged Stash PR RefSpec (described above), the other option is to do the merge locally as part of the build using the Jenkins git plugin (these only work for branches within the same repo):
+
+- In "Source Code Management" > "Git" > "Repositories" -> "Advanced" set "Refspec" to an empty value to enable the default behavior
+- Set the "Branch Specifier" to `*/${sourceBranch}`
```

Comment:
why not `/refs/heads/${sourceBranch}`

Reply:
I was thinking of it. I'll change it.

Actually, we can also change `+refs/pull-requests/*:refs/remotes/origin/pr/*` to `+refs/pull-requests/${pullRequestId}/*:refs/remotes/origin/${pullRequestId}/pr/*` in the suggested refspec.

Reply:
Killed the sourceBranch variant. It's not better than the `/from` variant, and the later is closer to the configurations recommended in the beginning of the README.md file.